CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing
+
+Contributions are welcome. For all non-trivial changes, please create an issue first to help us track what is being worked on. PR titles should follow the angular style for compatibility with our automated release tool. See [here](https://www.conventionalcommits.org/en/v1.0.0/) for an explanation of the allowable commit message format.
```

Comment:
The bit about PR title suggests that it will be used to trigger the release, but it actually doesn't